openpower/patches/mihawk-patches/occ/occ-20180525-fix-invalid-pwr-reading-when-no-raw-value.patch

```diff
@@ -0,0 +1,31 @@
+diff --git a/src/occ_405/amec/amec_sensors_power.c b/src/occ_405/amec/amec_sensors_power.c
+index bcd22c9..740cc08 100755
+--- a/src/occ_405/amec/amec_sensors_power.c
```

Comment:
Is the OCC team happy to support this patch either forever or accept it upstream? @wilbryan ?

Reply:
Yes and there should be a request to upstream the change

Reply:
How should I send the request, using github pull request or any other ways?

Reply:
I sent the PR to occ.
https://github.com/open-power/occ/pull/27